scripts/changelog.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Please note that this script only work with Github repositories.
+# Prerequisites: git, github cli
+
+GHCLI_BIN="ghcli"
+REPO="cloudskiff/driftctl"
+LATEST_TAG=$(git describe --abbrev=0) # Get the least created tag
+DEFAULT_BRANCH=$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
+BASE=$DEFAULT_BRANCH # Change this if you don't want to use the default branch as base
```

Comment:
```suggestion
BASE=origin/HEAD
```

Is there any reason we need to do this default branch logic? Can't we just log from `origin/HEAD`? If I understand the sed expression correctly, we're using the local pointer to `main`, rather than the remote-tracking origin/main or origin/HEAD updated by `git fetch`, and I'm not sure what the benefit of that is.